docs/rpc/openapi.yaml

```diff
@@ -486,6 +486,93 @@ paths:
           If tip == "latest", the query will be run from the latest known tip (includes unconfirmed state).
           If the tip is left unspecified, the stacks chain tip will be selected (only includes confirmed state).
 
+  /v2/clarity/marf/{clarity_marf_key}:
+    post:
+      summary: Get the MARF value for a given key
+      tags:
+        - Smart Contracts
+      operationId: get_clarity_marf_value
+      description: |
+        Attempt to fetch the value of a MARF key. The key is identified with [Clarity MARF Key].
```

Comment:
I think you can delete "The key is identified with [Clarity MARF Key]." since that is covered in the parameters.